<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpeederTubeGUI</class>
 <widget class="QWidget" name="SpeederTubeGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>820</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SpeederTube</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>600</width>
     <height>800</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Select the Spotify playlist you want to migrate:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
        <widget class="QComboBox" name="comboBox"/>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Create a new YouTube playlist to migrate to:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QRadioButton" name="radioButton">
      <property name="text">
       <string>Public</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QRadioButton" name="radioButton_2">
      <property name="text">
       <string>Private</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Or select the YouTube playlist you want to migrate to:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QComboBox" name="comboBox_2"/>
    </item>
    <item row="8" column="0" colspan="2">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="2">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>After you've selected the Spotify and YouTube playlists, click the button to start migrating:</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Start Migrating</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Stop Migrating</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0" colspan="2">
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="14" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Song(s) successfully migrated:</string>
      </property>
     </widget>
    </item>
    <item row="14" column="1">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Song(s) failed to migrate:</string>
      </property>
     </widget>
    </item>
    <item row="17" column="0">
     <widget class="QListView" name="listView">
     </widget>
    </item>
    <item row="17" column="1">
     <widget class="QListView" name="listView_2"/>
    </item>
    <item row="18" column="0" colspan="2">
     <widget class="Line" name="line_4">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="19" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Progress:</string>
      </property>
     </widget>
    </item>
    <item row="20" column="0" colspan="2">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item row="21" column="0" colspan="2">
     <widget class="QListView" name="listView_3"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
